### ISSUE TITLE
question about smplify fitting process code

### ISSUE THREAD

**Issue by dytan** [nkolot/SPIN#12]
Noticed the ['__ call__'](https://github.com/nkolot/SPIN/blob/master/smplify/smplify.py#L89) function of smplify fitting process not call 'loss.backward()', and the vars stayed unchange after the process, seems typo here.

**Comment by nkolot** [comment]
Yes, you are correct, I fixed it and pushed the changes. Thanks for pointing this out!
